applications/GeoMechanicsApplication/custom_elements/updated_lagrangian_U_Pw_diff_order_element.cpp

```diff
@@ -77,7 +77,7 @@ void UpdatedLagrangianUPwDiffOrderElement::CalculateAll(MatrixType&        rLeft
         this->CalculateIntegrationCoefficients(IntegrationPoints, Variables.detJuContainer);
     auto strain_vectors = StressStrainUtilities::CalculateStrains(
         deformation_gradients, b_matrices, Variables.DisplacementVector, Variables.UseHenckyStrain,
-        this->GetVoigtSize());
+        GetStressStatePolicy().GetVoigtSize());

```

Comment:
I think we might need a `this->` here, since it's a derived class
```suggestion
        this->GetStressStatePolicy().GetVoigtSize());
```

Reply:
added.